alcs-frontend/src/app/services/compliance-and-enforcement/documents/document.dto.ts

```diff
@@ -26,5 +26,6 @@ export interface UpdateComplianceAndEnforcementDocumentDto {
 
 export interface CreateComplianceAndEnforcementDocumentDto extends UpdateComplianceAndEnforcementDocumentDto {
   section?: Section;
+  chronologyEntryUuid?: string | null;
```

Comment:
C&E docs can now be optionally tied to a chronology entry. However, they are still linked to the file as well, so we can aggregate them later.